ci/check-signed-off-by.sh

```diff
@@ -12,17 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# this retrieves the merge commit created by GH
-parents=(`git log -n 1 --format=%p HEAD`)
+giturl=(`git config --get remote.origin.url`)
```

Comment:
I think you don't need to use parenthesis here, either use the back quote to enclose the command, or `$(...)`